.github/workflows/ci.yml

```diff
@@ -0,0 +1,65 @@
+name: CI
+on:
+  pull_request:
+    branches:
+      - master
```

Comment:
Not sure that's really the problem,  but you could try removing this line.

Reply:
looks like there is a widespread problem, It stopped working also on zygote https://github.com/FluxML/Zygote.jl/pull/973

Reply:
removing this line didn't help